src/modules/75-cd/components/PipelineSteps/Common/Terraform/Editview/InlineVarFile.tsx

```diff
@@ -90,7 +90,7 @@ const InlineVarFile = (props: InlineVarFileProps) => {
             })
           })}
         >
-          {formikProps => {
+          {(formikProps: any) => {
```

Comment:
Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/harness/harness-core-ui/pullRequest?prid=9232251)
```suggestion
          {(formikProps: unknown) => {
```